test/lib/BedrockTester.cpp

```diff
@@ -466,8 +466,8 @@ vector<SData> BedrockTester::executeWaitMultipleData(vector<SData> requests, int
                         }
                         break;
                     } else {
-                        // If it's been over 60s, give up.
-                        if (recvStart + 60'000'000 < STimeNow()) {
+                        // If it's been over 240s, give up.
+                        if (recvStart + 240'000'000 < STimeNow()) {
```

Comment:
This worked in 60s on my dev machine but not in travis, so I allowed it more time.